INSTALL

```diff
@@ -512,9 +512,9 @@
                    Build without support for the specified algorithm, where
                    <alg> is one of: bf, blake2, camellia, cast, chacha, cmac,
                    des, dh, dsa, ecdh, ecdsa, idea, md4, mdc2, ocb, poly1305,
-                   rc2, rc4, rmd160, scrypt, seed, siphash, sm4 or whirlpool.
-                   The "ripemd" algorithm is deprecated and if used is
-                   synonymous with rmd160.
+				   rc2, rc4, rmd160, scrypt, seed, siphash, sm3, sm4 or
+				   whirlpool.  The "ripemd" algorithm is deprecated and if used
+				   is synonymous with rmd160.
```

Comment:
Why was the indentation changed?  We prefer all spaces indentation as tabs can be set different.  You obviously use 4-column tabs ;-)

Reply:
Argh it was from a merge conflict between sm3 and sm4 😉 Thank you for this.